api/services/ocean/spark/parameters/query/clusterIdentifier.yaml

```diff
@@ -0,0 +1,7 @@
+in: query
+name: clusterIdentifier
```

Comment:
@thorsteinnth should this query param be renamed controllerClusterId in `wave-core` to match with the rest of the API?

Reply:
Will be implented in https://spotinst.atlassian.net/browse/BGD-1646